Business resume/resume.html

```diff
@@ -0,0 +1,143 @@
+    <link rel="stylesheet" href="homepage.css">
+    <link rel="stylesheet" href="resume.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Business Resume</title>
+</head>
+<body>
+    <header>
+        <div class="header-left">
+            <div class="header-box">
+                <div class="square">
+                </div>
+                <a href="index.html">Maya Nelson</a>
+            </div>
+            <span>/</span>
+            <a class="title-header" href="index.html">PROJECT MANAGER</a>
+        </div>
+
+        <div class="header-right">
+            <input id="toggle" type="checkbox"></input>
+
+            <label for="toggle" class="hamburger">
+            <div class="top-bun"></div>
+            <div class="meat"></div>
+            <div class="bottom-bun"></div>
+            </label>
+
+            <div class="nav">
+            <div class="nav-wrapper">
+            <nav>
+                <ul>
+                    <li><a href="index.html">About me</a></li>
+                    <li><a href="resume.html">Resume</a></li>
+                    <li><a href="#">Projects</a></li>
+                    <li><a href="contact.html">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+
+    <main>
+        <div class="title-main">
+            <div class="square">
+                    </div>
+            <h1>Resume</h1>
+        </div>
+
+        <div class="first-head-main">
+            <h2>Experience</h2>
+            <a href="#">DOWNLOAD CV</a>
+        </div>
+
+        <div class="resume-container">
+            <h4>year</h4>
+            <p>JOB POSITION</p>
+            <p>Company Name</p>
+            <p>Company Location</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iure nam minus debitis perferendis ipsa atque neque repudiandae illum assumenda quibusdam voluptates molestias laboriosam fugiat dolor sequi illo, aut veniam doloribus.</p>
+        </div>
+
+        <div class="second-head-main">
+            <h2>Education</h2>
+        </div>
+
+        <div class="resume-container">
+            <h4>year</h4>
+            <p>UNIVERSITY NAME</p>
+            <p>Degree Level</p>
+            <p>University Location</p>
+            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Iure nam minus debitis perferendis ipsa atque neque repudiandae illum assumenda quibusdam voluptates molestias laboriosam fugiat dolor sequi illo, aut veniam doloribus.</p>
+        </div>
+
+        <div class="resume-container-skills">
+            <h2>Professional skillset</h2>
+            <div class="skills">
+                <div class="skills-flex">
+                    <div class="square"></div>
```

Comment:
This list of skills could maybe be a `ul` list? It's possible to use an image (eg the blue square) as the list bullets